salesforce_sync.gemspec

```diff
@@ -19,11 +19,16 @@ Gem::Specification.new do |s|
   s.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   s.require_paths = ["lib"]
 
-  s.add_runtime_dependency "rails", [">= 3.0"]
+  s.add_runtime_dependency "rails", [">= 4.2.11"]
   s.add_runtime_dependency "delayed_job_active_record", [">= 4.1.1"]
   s.add_runtime_dependency "event_bus", [">= 1.1.1"]
   s.add_runtime_dependency "restforce", [">= 3.0.0"]
   s.add_runtime_dependency "salesforce_bulk_api", [">= 0.0.12"]
+  s.add_runtime_dependency "sprockets", [">= 2.12.5"]
```

Comment:
Hey @Shehbaz, I'm not sure, but I think we don't need to add these dependencies. 
Have you tried to run `bundle update` without adding these dependencies?

Reply:
I added those dependencies to be more explicit about the minimum versions that get installed. I have removed them now.